docker/caffe2/jenkins/common/install_rocm.sh

```diff
@@ -33,7 +33,6 @@ install_ubuntu() {
 
 install_centos() {
 
-  yum update -y
```

Comment:
Why?

Reply:
yum update was upgrading any centos 7.x image to centos 7.6. 
https://tecadmin.net/upgrade-centos-7-to-latest-release/